web_dashboard_tile/models/tile_category.py

```diff
@@ -0,0 +1,105 @@
+# © 2018 Iván Todorovich <ivan.todorovich@gmail.com>
+# © 2019-Today GRAP
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class TileCategory(models.Model):
+    _name = "tile.category"
+    _description = "Dashboard Tile Category"
+    _order = "sequence asc"
+
+    name = fields.Char(required=True)
+
+    sequence = fields.Integer(required=True, default=10)
+
+    active = fields.Boolean(default=True)
+
+    action_id = fields.Many2one(
+        string='Odoo Action', comodel_name='ir.actions.act_window',
+        readonly=True)
+
+    menu_id = fields.Many2one(
+        string='Odoo Menu', comodel_name='ir.ui.menu', readonly=True)
+
+    tile_ids = fields.One2many(
+        string='Tiles', comodel_name='tile.tile',
+        inverse_name='category_id')
+
+    tile_qty = fields.Integer(
+        string='Tiles Quantity',
+        compute='_compute_tile_qty',
+        store=True,
+    )
+
+    @api.depends('tile_ids')
+    def _compute_tile_qty(self):
+        for category in self:
+            category.tile_qty = len(category.tile_ids)
+
+    def _prepare_action(self):
+        self.ensure_one()
+        return {
+            'name': self.name,
+            'res_model': 'tile.tile',
+            'type': 'ir.actions.act_window',
+            'view_mode': 'kanban',
+            'domain': "["
+            "('hidden', '=', False),"
+            "'|', ('user_id', '=', False), ('user_id', '=', uid),"
+            "('category_id', '=', %d)"
+            "]" % self.id,
```

Comment:
```suggestion
            'domain': f"""
                [
                    ('hidden', '=', False),
                    '|', ('user_id', '=', False), ('user_id', '=', uid),
                    ('category_id', '=', {self.id})
                ]
            """,
```

This is probably more readable :)

Reply:
indeed ! thanks !

Reply:
FIY, I removed the "f" syntax, that was making travis failing. Otherwise LGTM.